app/views/user_mailer/send_flag_notification.text.erb

```diff
@@ -0,0 +1,8 @@
+Welcome to tfa.org
```

Comment:
I think this was left from the registration email.

Reply:
In this case, we need this view. One for text.erb and the other one html.erb as we have the other notifications.